scripts/generate_json_docs.py

```diff
@@ -290,10 +291,11 @@ def build_type(type_id, title, contents):
 
 def clean_source_path(source):
     source_path = ''
-    if 'gcloud-python' in source:
-        source_path = source.split('gcloud-python')[1]
+    for path in sys.path:
+        if path in source:
+            source_path = source.replace(path, '')
 
-    return source_path[1:]
+    return source_path
```

Comment:
That does seem simpler, given that we know we are looking only under that package.